package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fcp-fd-comms",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
Version has been updated to `0.2.0`. Regarding the release number, we currently have a [runbook](https://eaflood.atlassian.net/wiki/spaces/SFD/pages/5214076983/Single+Front+Door+Release+1.0) in place for the work that was done for a separate ticket ([SFDP-35 Messaging infrastructure](https://eaflood.atlassian.net/browse/SFDP-35)). The release number is listed as `1.0` - Fay and I were meant to double check the format for numbering releases but since the messaging infrastructure (Managed Identity & Service Bus) is down in the runbook as our first release, I assume this would be our second release? If so would the release be down as `1.0.1`? Let me know and I will update the current runbook + create a new one as a placeholder for this ticket.

Reply:
You don't need to align the release numbers in the release calendar to each microservice release.

Each release needs a number.  So just go 1, 2, 3, 4 etc no matter what version of what microservice we're releasing.  (this is included in the change request though)

Reply:
So 1.0.0 is fine

Even if the next release number is 50, it's fine for the next new feature added here to be 1.1.0

Reply:
Great, thank you for explaining. Will update the runbooks.